src/style.css

```diff
@@ -90,7 +90,7 @@ p {
 }
```

Comment:
![image](https://github.com/otmaneechchafyky/leaderboard/assets/93039359/4b76f766-7448-4942-b04d-e99b954437d6)

- [ ] It would be nice if you fix the above link which is in your Table of Contents to make your readme file descriptive and professional because right now this is broken :fearful: Kindly remove it if you don't have any section regarding this link :+1: